cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,42 @@
+import time
+from django.db import connections
+from django.db.utils import OperationalError
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    """Django command to pause execution until the database is available."""
+
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database...")
+        db_conn = None
+        attempts = 0
+        max_attempts = 30
+        wait_time = 1
+
+        while not db_conn:
+            try:
+                db_conn = connections["default"]
+                db_conn.cursor()
```

Comment:
The `connections["default"]` call could potentially raise a `KeyError` if the 'default' connection isn't defined. It might be better to use `connections.databases.get('default')` which will return `None` if 'default' isn't found.